### PR TITLE
Starboard

### PR DESCRIPTION
**Pull request by circuitsacul**
Don't gitpull after merge, I need to update the .env file with new info.

### CODE REVIEW

**Comment by CircuitSacul** [comment]
This is finished (as much as I'm going to do, anyways)

File: mcoding_bot/bot.py [hunk 1]
```diff
@@ -9,6 +9,9 @@
 
 from mcoding_bot.config import Config
 
+from .database import Database
```
Comment (mCodingLLC) [inline mcoding_bot/bot.py:12]: make absolute

Comment (CircuitSacul) [reply mcoding_bot/bot.py]: Done

File: mcoding_bot/database/__init__.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+"""Database for mCodingBot."""
+
+from .database import Database, Star, Message
```
Comment (mCodingLLC) [inline mcoding_bot/database/__init__.py:3]: make absolute

**Pull request merged**